examples/enclosure.tf

```diff
@@ -0,0 +1,22 @@
+provider "oneview" {
+	ov_username = "administrator"
+	ov_password = "madhav123"
+	ov_endpoint = "https://10.170.16.44"
```

Comment:
instead of actuall values, you can write <ov_username>, <ov_password>,<ov_instance_url>

Reply:
Replaced the actual values as suggested